src/RawAPI.js

```diff
@@ -193,7 +193,7 @@ export class RawAPI extends EventEmitter {
           },
           set(branch,modules,_hash){
             if(!_hash) _hash = Date.now()
-            else return self.req('POST',`${self.opts.apiPath}/user/code`, { branch, modules, _hash })
```

Comment:
The `else` was not in the base code, I added it because of a bad copy/paste.
If I don't remove it, the request is never done because `_hash` is undefined.